### DISCUSSION TITLE
Where can I find the last release of EthOS Simulator

### DISCUSSION THREAD

**Discussion by amaury10** [FrSkyRC/ETHOS-Feedback-Community#3329] [Q&A] [answered]
Hello guys,

I recently bought a Tandem X18 and I want to develop lua modules. The simulator seems to be a good way of testing.
The last release of it I cant find on github is 1.2. Is there any newer releases ? Why is it so difficult to find it ?

**Comment by bsongis-frsky** [comment, marked as answer]
Go to the latest release (today is 1.5.0 RC3 which should become 1.5.0 soon), open the assets and you will find the latest simulator

**Reply by amaury10** [reply]
Ok, thank you. Will the simulator always be released with every Ethos release ? If no, it could be a good idea to add a shortcut somewhere to find the last release of the simulator.

**Reply by FireflyJar** [reply]
Took me a moment to figure that out it is on github with the Radio's firmware,  they have to match to transfer model's files over.
